ddtrace/opentracer/tracer.py

```diff
@@ -71,24 +75,118 @@ def __init__(self, service_name=None, config=None, scope_manager=None):
 
     @property
     def scope_manager(self):
-        """"""
+        """Returns the scope manager being used by this tracer."""
         return self._scope_manager
 
     @property
     def active_span(self):
-        """"""
-        pass
+        """Gets the active span from the scope manager or none if it does not exist."""
+        scope = self._scope_manager.active
+        if scope is not None:
+            return scope.span
+        return None
```

Comment:
You could probably do a straight return of the result from `self._scope_manager.active` here as the `if` block is effectively a no-op (will only return None when scope is already None).

Reply:
Wait, but we still want to get the span attribute out of the scope. Is there a more pythonic way to write this?